.github/workflows/check-markdown-links.yml

```diff
@@ -0,0 +1,13 @@
+name: Check Markdown links
+
+on: push
+
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - uses: gaurav-nelson/github-action-markdown-link-check@v1
+      with:
+        use-quiet-mode: 'yes'
+        check-modified-files-only: 'yes'
```

Comment:
can we use https://github.com/tcort/markdown-link-check directly, and add this to our docs-website linter?

don't think we need another github workflow for this